.github/workflows/pr-verify.yaml

```diff
@@ -5,15 +5,64 @@ on:
     types: [opened, edited, synchronize, reopened]
 
 permissions:
-  checks: write # Allow access to checks to write check runs.
+  contents: read
+  pull-requests: write
```

Comment:
I would prefer if we only log those messages in the actions and not write to the PR

I think adding the comments to the PR will lead to a lot of noise

(Prow usually implements some cleanup to delete comments after issues are fixed, but I don't want to do that here)

Also just a lot better if we require less permissions for security reasons

Reply:
Okay, I make it not to write to the PR.